volume/drivers/extpoint_test.go

```diff
@@ -21,3 +22,79 @@ func TestGetDriver(t *testing.T) {
 		t.Fatalf("Expected fake driver, got %s\n", d.Name())
 	}
 }
+
+type tpms struct {
+	bind      string
+	driver    string
+	expDest   string
+	expSource string
+	expName   string
+	expDriver string
+	expRW     bool
+	fail      bool
+}
+
+func TestParseMountSpecSplit(t *testing.T) {
+	var cases []tpms
+	if runtime.GOOS != "windows" {
```

Comment:
can we switch this around to `runtime.GOOS == "windows"`? I find this generally easier to read.